.travis.yml

```diff
@@ -4,6 +4,6 @@ node_js:
   - "6"
   - "5"
   - "4"
-after_script:
-  - npm run coveralls
+# after_script:
+#   - npm run coveralls
```

Comment:
Because at the moment all the test are failing, because of the `reply.reply`, once we decide how handle this, I'll update all the test :)